vitrina/datasets/forms.py

```diff
@@ -363,17 +363,28 @@ class Meta:
             "managed_by_publisher",
             "landing_page",
             "parent",
+            "service_type"
         )
 
     def __init__(self, request=None, organization=None, *args, **kwargs):
         super().__init__(request, organization, *args, **kwargs)
 
+        self.fields["service_type"] = ModelMultipleChoiceField(
+            queryset=Concept.objects.filter(
+                concept_schemas__uri="http://publications.europa.eu/resource/authority/data-service-type"
+            ).distinct(),
+            required=False,
+            widget=forms.SelectMultiple(attrs={'size': '3'}),
+            label=_("Tipas")
+        )
+
```

Comment:
Whole `ModelMultipleChoiceField` doesn't have to be overridden, if only queryset (and maybe widget) needs to be changed. Label, required, etc. are automatically used from model.

We should not hardcode the URI. You can check how `INFORMATION_SYSTEM_IMPORTANCE_SCHEMA_URI` is implemented


Or is there another reason to override whole field?

```suggestion
        self.fields["service_type"].queryset = Concept.objects.filter(
            concept_schemas__uri=Dataset.SYSTEM_TYPE_SCHEME_URI
        ).distinct()
```

Reply:
"We should not hardcode the URI. You can check how INFORMATION_SYSTEM_IMPORTANCE_SCHEMA_URI is implemented"

You mean in this queryset filter it should not be hardcoded or when creating the Concept_schema in the migration, the URI should be dct:type and the same here?

Reply:
In migrations, I think it's better to hardcode instead of importing variable. Elsewhere - it's better to use it from imported variable

Reply:
Changed according to the suggestion, did not know that it was not necessary to specify that the field will be a multiple choice field. Thanks